From: Jo-Philipp Wich <jow@openwrt.org>
Date: Wed, 2 Dec 2015 16:10:09 +0000
Subject: tools: make-ext4fs: explicitely request C collation for sorting

Do not rely on the system default collaction when sorting directory entries
but explicitely request "C" collation.

Signed-off-by: Jo-Philipp Wich <jow@openwrt.org>

SVN-Revision: 47689

diff --git a/tools/make-ext4fs/Makefile b/tools/make-ext4fs/Makefile
index d6858cdd97a537395dc909ecaa8fde025c43dc85..ba988290879ced9709e6337b892a12329727c3ee 100644
--- a/tools/make-ext4fs/Makefile
+++ b/tools/make-ext4fs/Makefile
@@ -7,14 +7,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=make-ext4fs
-PKG_VERSION:=2015-09-14
+PKG_VERSION:=2015-12-02
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://git.openwrt.org/project/make_ext4fs.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=98e3d5c1ceddd4a32b92c1cfafe929fe006fb98c
+PKG_SOURCE_VERSION:=bd53eaafbc2a89a57b8adda38f53098a431fa8f4
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_MIRROR_MD5SUM:=c736e546de3b36da73f5e84a9ce3ffee
+PKG_MIRROR_MD5SUM:=742ed45a833c81de5732a3adb827cde5
 PKG_CAT:=zcat
 
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)-$(PKG_VERSION)
